src/hooks/useQueryLogs.ts

```diff
@@ -88,24 +86,28 @@ export const useQueryLogs = () => {
 			setError(null);
 
 			const logsQueryRes = isQuerySearchActive
-				? await getQueryResult({ ...logsQuery, access: [] }, appendOffsetToQuery(custSearchQuery, logsQuery.pageOffset))
-				: await getQueryLogs(logsQuery);
+				? await getQueryResult(
+						{ ...logsQuery, access: [] },
+						appendOffsetToQuery(custSearchQuery, logsQuery.pageOffset),
+						{ fields: 'true' },
+				  )
+				: await getQueryLogs(logsQuery, { fields: 'true' });
 
-			const data = logsQueryRes.data;
+			const data = logsQueryRes.data.records;
 
 			if (logsQueryRes.status === StatusCodes.OK) {
 				const jqFilteredData = isJqSearch(instantSearchValue) ? await jqSearch(data, instantSearchValue) : [];
-				return setLogsStore((store) => setData(store, data, schema, jqFilteredData));
+				return setLogsStore((store) => setData(store, data, logsQueryRes.data, jqFilteredData));
 			}
 			if (typeof data === 'string' && data.includes('Stream is not initialized yet')) {
-				return setLogsStore((store) => setData(store, [], schema));
+				return setLogsStore((store) => setData(store, [], logsQueryRes.data));
```

Comment:
Why are you trying to set the data when the stream is not initialized ?
I don't think this `logsQueryRes.data` will hold what you expect in this.